.dockerignore

```diff
@@ -0,0 +1,2 @@
+Dockerfile
+docker-compose.yml
```

Comment:
I think it's ok that the image contains the two files that you're ignoring.

Instead I think you want to exclude temporary files, e.g.:

```
*.egg-info
.coverage
.git
.tox
build
coverage-html
docs/_site
venv
.tox
**/__pycache__
*.pyc
```